Px.Utils/Validation/SyntaxValidation/SyntaxValidationUtilityMethods.cs

```diff
@@ -202,14 +207,24 @@ internal static bool ValueLineChangesAreCompliant(string input, PxFileSyntaxConf
         /// </summary>
         /// <param name="input">The input string to clean</param>
         /// <param name="syntaxConf">The syntax configuration for the PX file. The syntax configuration is represented by a <see cref="PxFileSyntaxConf"/> object.</param>
+        /// <param name="forceful">A boolean that can be set true if the cleaning should be done forcefully</param>
         /// <returns>Returns a string that is the input string cleaned from new line characters and quotation marks</returns>
-        internal static string CleanString(string input, PxFileSyntaxConf syntaxConf)
+        internal static string CleanString(string input, PxFileSyntaxConf syntaxConf, bool forceful = false)
```

Comment:
I suggest moving the forceful section to a separate function or to inline it in to the caller. The goal is to know what the function does just by the signature, now this is bit obscure.